animalai/arenas/GoodGoal_Fixed.yml

```diff
@@ -7,11 +7,11 @@ arenas:
       - !Item
         name: Agent
         positions:
-          - !Vector3 { x: 5, y: 1, z: 5 }
+          - !Vector3 { x: 5, y: 0, z: 5 }
```

Comment:
Just to let you know the y value constraints are corrected in unity (as heads up note)

Reply:
Good change.